snap/snapcraft.yaml

```diff
@@ -277,7 +277,6 @@ parts:
 
       # Add bash completion for microk8s kubectl.
       bins/kubectl completion bash | sed "s/complete -o default -F __start_kubectl kubectl/complete -o default -F __start_kubectl microk8s kubectl/g" | sed "s/complete -o default -o nospace -F __start_kubectl kubectl/complete -o default -o nospace -F __start_kubectl kubectl/g" > kubectl.bash
-      bins/kubectl completion bash | sed "s/complete -o default -F __start_kubectl kubectl/complete -o default -F __start_kubectl microk8s.kubectl/g" | sed "s/complete -o default -o nospace -F __start_kubectl kubectl/complete -o default -o nospace -F __start_kubectl kubectl/g" > kubectl.bash
```

Comment:
This is not a duplicate line. There are two commands for wich we want to have auto completion: `microk8s.kubectl` and `microk8s kubectl`.